src/components/TextArea/TextArea.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { HTMLInputTypeAttribute } from "react";
+import "./textarea.css";
+
+export interface TextAreaProps {
+  /**
+   * The id of the textarea element. Use this prop to make value and placeholder accessible for screen readers.
+   */
+  id?: string;
```

Comment:
WDYM kaya make value and placeholder accessible ? I think id in this case g bakal kepake d since lu bisa pass key, sama by right id nya g boleh di modify sama css lain